apps/vanilla/src/pages/index.html

```diff
@@ -1,16 +1,160 @@
+					<h2 class="category__books-header">
+						<span class="subtitle-font-semibold letter-spacing-tiny category__books-header-year">2020</span>
+						<span class="subtitle-font-semibold letter-spacing-tiny">YEAR</span>
+					</h2>
+
+					<div class="category__books-body">
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-1.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Refactoring UI
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-2.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Master the Skill of Debugging CSS
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-3.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Blue Period
+							</h3>
+						</article>
+					</div>
+				</section>
+
+				<section class="category__books">
+					<h2 class="category__books-header">
+						<span class="subtitle-font-semibold letter-spacing-tiny category__books-header-year">1980</span>
+						<span class="subtitle-font-semibold letter-spacing-tiny">YEAR</span>
+					</h2>
+
+					<div class="category__books-body">
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-4.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Yokohama Kaidashi Kikou
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-5.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Learning JavaScript Design Patterns
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-6.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Yotsuba to!
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-7.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Domain-Driven Design
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-8.png" alt="Book image">
+							<h3 class="body-font-small two-line-text category__book-title">
+								Implementing Domain-Driven Design
+							</h3>
+						</article>
+
+						<article class="category__book">
+							<img class="category__book-image" src="/assets/images/books/image-9.png" alt="Book image">
```

Comment:
```suggestion
							<img class="category__book-image" src="/assets/images/books/image-9.png" alt="Book image">
```
The `alt` attribute is used not only for alternative display of the image but also for description. 

For example, a user with visual impairments will only hear the text from the alt attribute.

Ideally, you should describe what is in the picture in this attribute. It will be enough for the user to understand that this is a book poster and recognize its title